src/lib/split-view/split-view-panel/split-view-panel.html

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div class="forge-split-view-panel" id="root" part="root">
+    <div class="forge-split-view-panel__handle" id="handle" part="handle" role="separator" aria-controls="content" tabindex="0">
+      <forge-icon class="forge-split-view-panel__icon" id="icon" part="icon"></forge-icon>
+      <forge-ripple part="ripple"></forge-ripple>
+    </div>
+    <div class="forge-split-view-panel__content" id="content" part="content" role="group">
```

Comment:
I added a `group` role to the content element. This *might* be misused here but seems to do what I want — exposes it to the accessibility tree as a logical grouping of a larger widget without adding it to the page summary.